.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+
+<!-- A short (or detailed) description of what this PR does and why these changes are needed -->
+
+## How to Test
+
+<!--- Please describe in detail how you tested your changes -->
+
+## Checklist
+
+- [ ] I have performed a self-review of my code.
+- [ ] I have added tests for my changes.
+- [ ] I have commented on my code, particularly in hard-to-understand areas.
```

Comment:
should we run two more items for lints and tests ?

```
- [ ] I have run linter locally (`npm run lint`) and all checks pass.
- [ ] I have run tests locally (`npm test`) and all tests pass.
```

Reply:
Good idea!